etcdserver/server.go

```diff
@@ -153,13 +153,17 @@ type Server interface {
 
 // EtcdServer is the production implementation of the Server interface
 type EtcdServer struct {
-	// r and inflightSnapshots must be the first elements to keep 64-bit alignment for atomic
-	// access to fields
-
-	// count the number of inflight snapshots.
-	// MUST use atomic operation to access this field.
+	// inflightSnapshots count the number of inflight snapshots.
+	// MUST use atomic operation to access, keep 64bit aligned.
 	inflightSnapshots int64
-	Cfg               *ServerConfig
+	// appliedIndex MUST use atomic operation to access, keep 64bit aligned.
+	appliedIndex uint64
```

Comment:
probably shouldn't be a godoc,
`appliedIndex uint64 // must be 64-bit aligned for atomic operations`?